src/Main.java

```diff
@@ -0,0 +1,110 @@
+import org.json.JSONArray;
+import org.json.JSONObject;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class Main {
+    public static void main(String[] args) {
+        try {
+            // Read the contents of the file as a string
+            String jsonStr = new String(Files.readAllBytes(Paths.get("src/main/resources/stock_data.json")));
+
+            // Create a JSONArray from a string
+            JSONArray jsonArray = new JSONArray(jsonStr);
```

Comment:
please use Jackson or Gson libraries for json parsing  as they are used on the 99% of the prod code.